API-Samples/utils/util_init.cpp

```diff
@@ -793,10 +798,18 @@ void init_swapchain_extension(struct sample_info &info) {
     // the surface has no preferred format.  Otherwise, at least one
```

Comment:
Not your problem, but I don't think this comment is true anymore

Reply:
The issue is that we no longer need to check for VK_FORMAT_UNDEFINED as the only entry?  Yeah, I can do that.  Ping me if I've got the wrong idea.

Reply:
Yes, it's now illegal to return VK_FORMAT_UNDEFINED